flood_adapt/api/output.py

```diff
@@ -43,6 +43,45 @@ def get_roads(name: str, database: IDatabase) -> gpd.GeoDataFrame:
     return database.get_roads(name)
 
 
+def get_obs_point_timeseries(name: str, database: IDatabase) -> gpd.GeoDataFrame:
+    """Return the HTML strings of the water level timeseries for the given scenario.
+
+    Parameters
+    ----------
+    name : str
+        The name of the scenario.
+    database : IDatabase
+        The database object.
+
+    Returns
+    -------
+    str
+        The HTML strings of the water level timeseries
+    """
+    # Get the direct_impacts objects from the scenario
+    hazard = database.get_scenario(name).direct_impacts.hazard
+
+    # Check if the scenario has run
+    if not hazard.sfincs_has_run_check():
+        raise ValueError(
+            f"Scenario {name} has not been run. Please run the scenario first."
+        )
+
+    database_path = Path(database.input_path).parent
+    output_path = database_path.joinpath("output", "Scenarios", hazard.name)
+    gdf = database.get_obs_points()
+    timeseries_output_path = []
```

Comment:
It's a bit cleaner to do this as a list comprehension:
gdf["html"] = [ str(output_path.joinpath("Flooding", f"{station}_timeseries.html")) for station in gdf.name ]

Reply:
Done